### PR TITLE
Updated information around the push proxy

### PR DESCRIPTION
**Pull request by jasmith-lp**
See DX-108

### CODE REVIEW

File: pages/Documents/MessagingChannels/PushNotificationService/Configuration/push-proxy.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ permalink: push-notification-service-configuration-push-proxy.html
 indicator: messaging
 ---
 
+![](img/pusher/PushProxyFlow.png)
```
Comment (j9t) [inline pages/Documents/MessagingChannels/PushNotificationService/Configuration/push-proxy.md:17]: Please add a text alternative (“alt text”). (If possible, for other images on the page, too.)

File: pages/Documents/MessagingChannels/PushNotificationService/Configuration/push-proxy.md [hunk 2]
```diff
@@ -30,17 +32,24 @@ In order to enable the push proxy feature, you will require the following data a
...
+#### Basic Auth
+
+* HTTP Basic Authentication (BA) header.
 
     * Username. _Example_:  Alibaba.
 
     * Password. _Example_: opensesame.
 
+
+#### Mutual TLS ([TLS Authentication](../tls-authentication.md))
```
Comment (j9t) [inline pages/Documents/MessagingChannels/PushNotificationService/Configuration/push-proxy.md:44]: Does this link work? Consider linking to the “permalink” (it seems to be `(push-notification-service-tls-authentication.html)`).

File: pages/Documents/MessagingChannels/PushNotificationService/overview.md [hunk 3]
```diff
@@ -21,14 +21,5 @@ Mobile App Messaging provides brands with the ability to provide their customers
...
-
-1. **Retrieve your domain**. Use the [LivePerson Domain API](agent-domain-domain-api.html) to retrieve this information by providing the following service name:
-
-	* pusher
-
-3. Note the [API terms of use](https://www.liveperson.com/policies/apitou).
-
-4. When using this API, it is recommended that you implement our [Retry Policy and KeepAlive best practices](guides-retry-policy.html).
+LivePerson’s push service interfaces directly with Apple’s APNS and Google’s FCM push notification services. For brands who wish to refrain from sharing their tokens/certificates, LivePerson offers the option to configure their push to the URIs of their choice and to utilize their own logic, see:
+[Push Proxy](Configuration/push-proxy.md)
```
Comment (j9t) [inline pages/Documents/MessagingChannels/PushNotificationService/overview.md:25]: Please remove the line break, or add another line break.

Also, please check on the link.

File: pages/Documents/MessagingChannels/PushNotificationService/tls-authentication.md [hunk 4]
```diff
@@ -12,7 +12,8 @@ permalink: push-notification-service-tls-authentication.html
 indicator: messaging
 ---
 
-Customers supply a valid client key and certificate for a TLS connection (one-way SSL). The key and certificate is used in every request to the relevant proxy call. The key is securely stored in LivePerson’s server.
+Customers supply a valid client key and certificate for an mutual TLS connection (two-way SSL).
+The key and certificate is used in every request to the relevant proxy call. The key is securely stored in LivePerson’s server.
```
Comment (j9t) [inline pages/Documents/MessagingChannels/PushNotificationService/tls-authentication.md:16]: If this is a new paragraph, please add an additional line break.

**Review by j9t — CHANGES_REQUESTED** [review]
Minor things—thanks @jasmith-lp!

**Review by Ghasfj — APPROVED** [review]

File: pages/Documents/MessagingChannels/PushNotificationService/tls-authentication.md [hunk 5]
```diff
@@ -12,7 +12,8 @@ permalink: push-notification-service-tls-authentication.html
 indicator: messaging
 ---
 
-Customers supply a valid client key and certificate for a TLS connection (one-way SSL). The key and certificate is used in every request to the relevant proxy call. The key is securely stored in LivePerson’s server.
+Customers supply a valid client key and certificate for an mutual TLS connection (two-way SSL).  
+The key and certificate is used in every request to the relevant proxy call. The key is securely stored in LivePerson’s server.
```
Comment (j9t) [inline pages/Documents/MessagingChannels/PushNotificationService/tls-authentication.md:16]: Please either add or remove a line break between lines 15 and 16, depending on whether these should constitute 1 or 2 paragraphs.

**Review by j9t — CHANGES_REQUESTED** [review]

File: pages/Documents/MessagingChannels/PushNotificationService/tls-authentication.md (hunk 5 above)
Comment (jasmith-lp) [reply pages/Documents/MessagingChannels/PushNotificationService/tls-authentication.md:16]: Hey @j9t in markdown a line break is a double space before the end of the line. (which I added).
I've noticed a lot of trending towards using HTML as a supplementary syntax (eg the image links). What is the standard here?

Comment (j9t) [reply pages/Documents/MessagingChannels/PushNotificationService/tls-authentication.md:16]: That’s not visible here, and not even in the “raw” file.

Could you please take either approach, have a blank line between these if they are to mean 2 paragraphs, or remove the line break entirely, if this to be 1 paragraph? Thank you!

**Review by j9t — CHANGES_REQUESTED** [review]
> I've noticed a lot of trending towards using HTML as a supplementary syntax (eg the image links). What is the standard here?

The standard is to use Markdown wherever possible (see the new [“content and code” style guide section](https://github.com/LivePersonInc/developer-center#notes-on-content-and-code) in the README).

We’re going to add to these guidelines and are welcoming feedback. We’ll also follow up again internally to promote these.

File: pages/Documents/MessagingChannels/PushNotificationService/overview.md [hunk 6]
```diff
@@ -19,16 +19,7 @@ indicator: messaging
...
-
-1. **Retrieve your domain**. Use the [LivePerson Domain API](agent-domain-domain-api.html) to retrieve this information by providing the following service name:
-
-	* pusher
-
-3. Note the [API terms of use](https://www.liveperson.com/policies/apitou).
-
-4. When using this API, it is recommended that you implement our [Retry Policy and KeepAlive best practices](guides-retry-policy.html).
+LivePerson’s push service interfaces directly with Apple’s APNS and Google’s FCM push notification services. For brands who wish to refrain from sharing their tokens/certificates, LivePerson offers the option to configure their push to the URIs of their choice and to utilize their own logic, see:  
+[Push Proxy](push-notification-service-configuration-push-proxy.html)
```
Comment (j9t) [inline pages/Documents/MessagingChannels/PushNotificationService/overview.md:25]: Please add an extra line break between paragraph and image (i.e., there should be one blank line between the two).

**Review by j9t — APPROVED** [review]
Approving, but please address the note for pages/Documents/MessagingChannels/PushNotificationService/overview.md. Thanks.

File: pages/Documents/MessagingChannels/PushNotificationService/overview.md (hunk 6 above)
Comment (jasmith-lp) [reply pages/Documents/MessagingChannels/PushNotificationService/overview.md:25]: added

**Pull request merged**